tools/llm_bench/llm_bench_utils/config_class.py

```diff
@@ -138,7 +138,7 @@ class UseCaseTextToSpeech(UseCase):
                                  "tiny-llama", "tinyllama", "opt", "opt-", "pythia", "pythia-", "stablelm", "stablelm-", "stable-zephyr-", "rocket-",
                                  "vicuna", "dolly", "bloom", "red-pajama", "xgen", "longchat", "jais", "orca-mini", "baichuan", "qwen", "zephyr",
                                  "mistral", "mixtral", "phi", "phi2-", "minicpm", "gemma", "deci", "phi3", "internlm", "olmo", "starcoder", "instruct-gpt",
-                                 "granite", "granitemoe", "gptj", "yi-"]),
+                                 "granite", "granitemoe", "gptj", "yi-", "exaone"]),
```

Comment:
Also, it needs to update documentation. Add tests for this model on tiny model, etc.